lib/src/utils/cached_profile_information.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:matrix/matrix_api_lite.dart';
+
+class CachedProfileInformation extends ProfileInformation {
```

Comment:
The name `CachedProfileInformation` can be misleading here

Reply:
I will reply here if I have a better name suggestion but do you think extending `ProfileInformation` for the existing `User` class is a good idea? Could be a mixin too

Reply:
No, the User class is an extension to the StrippedStateEvent and this is definitely not what we have. I chose `CachedProfileInformation` because we already introduced `CachedPresence` for a `Presence` which we have cached